index.html

```diff
@@ -7,12 +7,143 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
-    <link href="index.css" rel="stylesheet";
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://unicons.iconscout.com/release/v4.0.8/css/line.css">
+    <link href="index.css" rel="stylesheet">
     <title>Junior Achievers Kenya</title>
 
 </head>
 
 <body>
+
+    <header>
+                   
+        
+        <nav>
+            
+            <div class="top">
+                <a href="#"><i class="uil uil-facebook-f"></i></a>
+                <a href="#"><i class="uil uil-twitter"></i></a>
+                <a href="#">English</a>
+                <a href="#">My Page</a>
+                <a href="#">Logout</a>
+    
+                </div>
+                
+            <div class="bottom">
+            <a href="index.html" class="logo">JA KENYA</a>
+
+            <ul class="nav-menu">
+                <li class="nav-item"><a href="about.html" class="nav-link">About</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Sponsor</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+                <li><a href="#" class="nav-link">CC Campaign</a></li>
+                
+            </ul>
+
+           
+
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+
+        </div>
+        </nav>
+        
+    </header>
+
+    <section class="hero">
+        <div class="hero-wrapper">
+        <p class="hello">"Hello Sharing world</p>
+        <h1>CREATIVE COMMONS <br /> GLOBAL SUMMIT <br />2015</h1>
+        <p class="description">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea.</p>
```

Comment:
- [ ] I think you should consider reducing the width of your introductory paragraph to match the design given to you 👍🏽 

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235220499-f51154fe-d2c5-448e-910d-5b146902fcd2.png)

design guideline 👇🏽
![image](https://user-images.githubusercontent.com/63473584/235220372-9c25986b-0188-459b-8473-42461ad4ca63.png)